README.md

```
@@ -31,118 +31,25 @@ Features
-``` {.sourceCode .bash}
-$ pip install proxybroker
-```
-
-To install the latest development version from GitHub:
-
-``` {.sourceCode .bash}
-$ pip install -U git+https://github.com/bluet/proxybroker2.git
-```
+> Pypi package in progress. We will upload the up-to-date package under new name (proxybroker2) when the support for 3.10 is ready. https://github.com/bluet/proxybroker2/issues/89
 
 ### Use pre-built Docker image
```

Markdownlint (0.35.0)

Expected: setext; Actual: atx
Heading style

(MD003, heading-style)